class UserNotFoundException(Exception):
    pass
class UserAccessInvalidException(Exception):
    pass

